inductiva/_cli/cmd_storage/remove.py

```diff
@@ -1,20 +1,35 @@
 """Remove the user's remote storage contents via CLI."""
 
+import sys
+
 from inductiva import storage
 
 
 def remove(args):
     """Remove user's remote storage contents."""
-    path = args.path
+    paths = args.path
     confirm = args.confirm
+    all_paths = args.all
+
+    if paths and all_paths:
+        print("inductiva storage remove: error: "
+              "argument path not allowed with argument --all")
+        sys.exit(1)
 
     if not confirm:
-        prompt = input(f"Are you sure you want to remove {path}? (y/n)")
+        if all_paths:
+            prompt = input("Are you sure you want to remove everything "
+                           "from your storage(y/[N])")
+        else:
+            prompt = input(f"Are you sure you want to remove {len(paths)}"
+                           " path(s)? (y/[N])")
         confirm = prompt.lower() in ["y", "ye", "yes"]
 
     if confirm:
-        print("Removing %s in the remote storage", path)
-        storage.rmdir(path, confirm=confirm)
+        if all_paths:
+            storage.rmdir("*", confirm=confirm)
+        for path in paths:
+            storage.rmdir(path, confirm=confirm)
```

Comment:
Just to be extra explicit:

```suggestion
            storage.rmdir("*", confirm=True)
        for path in paths:
            storage.rmdir(path, confirm=True)
```